paddle/fluid/framework/operator.h

```diff
@@ -386,6 +386,8 @@ class ExecutionContext {
     return *boost::get<std::shared_ptr<T>>((*kernel_configs_)[idx]);
   }
 
+  const RuntimeContext Context() const { return ctx_; }
```

Comment:
`const RuntimeContext &Context() const { return ctx_; }` to avoid copy.

Reply:
done